bors.toml

```diff
@@ -0,0 +1,5 @@
+status = ["ci/circleci"]
+# Uncomment this to use a two hour timeout.
+# The default is one hour.
+# timeout_sec = 7200
+# required_approvals = 2
```

Comment:
Why uncomment `required_approvals`,  we need this check in case of  unexpected careless merging.

Reply:
yeah. I know that, still try to use it.